src/tests/conformance/query/get_by_hash.rs

```diff
@@ -74,3 +75,49 @@ async fn should_get_transaction_by_hash() {
     assert!(synth_node.expect_message(&check).await);
     node.stop().expect("unable to stop stateful node");
 }
+
+#[tokio::test]
+async fn should_query_for_transactions_after_have_transactions() {
+    // ZG-CONFORMANCE-009
+    // Create stateful node.
+    let target = TempDir::new().expect("unable to create TempDir");
+    let mut node = build_stateful_builder(target.path().to_path_buf())
+        .expect("unable to get stateful builder")
+        .start(false)
+        .await
+        .expect("unable to start stateful node");
+
+    // Wait for correct state and account data.
+    wait_for_state("proposing".into()).await;
+    let account_data = wait_for_account_data("rNGknFCRBZguXcPqC63k6xTZnonSe6ZuWt") // TODO make constant
+        .await
+        .expect("unable to get account data");
+    let tx = account_data.result.account_data.previous_transaction;
+
+    // Inform about transaction via peer protocol.
+    let mut tx_hash = [0u8; 32];
+    hex::decode_to_slice(&tx, &mut tx_hash as &mut [u8])
```

Comment:
```suggestion
    hex::decode_to_slice(&tx, &mut tx_hash[..])
```
haven't tested it, but this should work

Reply:
Done